src/commands/basics.js

```diff
@@ -10,11 +10,13 @@ const { data } = require("../jsons"),
  * @param {string} text The context of the message that should be sent.
  */
 function message(bot, msg, text) {
+  var data_options = JSON.parse(JSON.stringify(data.settings.messageOptions));
```

Comment:
Quick & dirty, ma capisco il non voler importare un pacchetto intero solo per una copia profonda (JavaScript per qualche motivo a oggi ancora non la supporta built-in). Perlopiù stiamo per riscrivere tutto in Go in ogni caso.